src/main/java/frc/robot/commands/StorageID.java

```diff
@@ -0,0 +1,72 @@
+package frc.robot.commands;
+
```

Comment:
The filename is fine, but I'm not sure what StorageID means, perhaps ManageStorage is a more meaningful name?